provisioner/ibm-s3fs-provisioner.go

```diff
@@ -226,6 +262,12 @@ func (p *IBMS3fsProvisioner) Provision(options controller.VolumeOptions) (*v1.Pe
 		return nil, fmt.Errorf(pvcName+":"+clusterID+":invalid value for auto-delete-bucket, expects true/false: %v", err)
 	}
 
+	if pvc.SetAccessPolicy == "" {
+		pvc.SetAccessPolicy = "true"
```

Comment:
I think we dont need to set bucketPolicy equals to  `true` if it is not passed in PVC spec. The code should make decision based on the value set during plugin deployment.

Reply:
Agree. removed setting SetAccessPolicy to true.